tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCompactNumbers.java

```diff
@@ -13,15 +13,19 @@
 import org.unicode.cldr.util.ICUServiceBuilder;
 import org.unicode.cldr.util.StandardCodes;
 import org.unicode.cldr.util.SupplementalDataInfo;
+import org.unicode.cldr.util.SupplementalDataInfo.PluralInfo;
+import org.unicode.cldr.util.SupplementalDataInfo.PluralInfo.Count;
 import org.unicode.cldr.util.VerifyCompactNumbers;
 
 import com.ibm.icu.text.CompactDecimalFormat;
 import com.ibm.icu.text.CompactDecimalFormat.CompactStyle;
 import com.ibm.icu.text.NumberFormat;
+import com.ibm.icu.text.PluralRules.FixedDecimal;
+import com.ibm.icu.util.Currency;
 import com.ibm.icu.util.ULocale;
 
 public class TestCompactNumbers  extends TestFmwkPlus {
-    static final boolean DEBUG = false;
+    static final boolean DEBUG = true;
```

Comment:
is it intentional and safe to make DEBUG = true in production?

Reply:
I don't see any problem with it, just checking it's intentional; it's in a unit test anyway